packages/database/src/migrations/20240818212910-removeMapOverlayConfigFromPgNotify-modifies-schema.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const scrubMapOverlayConfig = `
+CREATE OR REPLACE FUNCTION public.scrub_map_overlay_config(current_record jsonb DEFAULT NULL::jsonb, tg_table_name name DEFAULT NULL::name)
+ RETURNS json
+ LANGUAGE plpgsql
+AS $function$
+    BEGIN
+      IF TG_TABLE_NAME != 'map_overlay' THEN
+        RETURN current_record;
+      END IF;
+      IF current_record IS NULL THEN
```

Comment:
```suggestion
      ELSIF current_record IS NULL THEN
```
Looked it up cos I was curious, and I think this would work according to postgres docs if ya wanted: https://www.postgresql.org/docs/7.2/plpgsql-control-structures.html